src/officehours_ui/templates/base.html

```diff
@@ -3,41 +3,48 @@
 {% get_flatpages as flatpages %}
 <!doctype html>
 <html lang="en">
-  <head>
+
+<head>
```

Comment:
I reviewed and am okay with the formatting changes made. It seems more readable and I didn't see any extra changes or new issues.

Reply:
It makes it really hard to see what's changed with the formatting changes mixed in. This file is a mixture of tabs and spaces so probably should be re-formatted and re-indented.

I'd personally leave these as-is and if we want to change the indentation tabs/spaces that would be on a different issue. Almost every editor you can configure to only adjust where you actually edit. For instance VSCode will detect and keep indentation as-is. So not sure what editor was used here. I'm going to push a commit to revert the whitespace and try to leave in the changes.